### DISCUSSION TITLE
mobile queries

### DISCUSSION THREAD

**Discussion by kethamanasa** [nolimits4web/swiper#5475] [General]
Hai.. 
          How  to adjust swiper scroll in mobile view.    I added swiper scroll in the about page of our website. But i wanted to adjust it in mobile view that only one card should be scrolled in the mobile view.

**Comment by shafschwd** [comment]
breakpoints: {
        320: {
          slidesPerView: 1,
          spaceBetween: 20,
        },
      }
Like this

**Comment by Arhell** [comment]
https://swiperjs.com/swiper-api#param-breakpoints
